source/data_api.html.md

```diff
@@ -37,19 +55,82 @@ def get_token(client_id, client_secret, scope, endpoint, audience):
   token = resp.json()['access_token']
   return token
 
-token = get_token('<client_id>',
-                  '<client_secret>',
-                  'all',
-                  'https://api.welkinhealth.com/v1/token',
-                  'https://api.welkinhealth.com/v1/token')
+token = get_welkin_api_token('<client_id>',
+                             '<client_secret>',
+                             'all',
+                             'https://api.welkinhealth.com/v1/token')
+```
+
+```javascript
+const axios = require('axios');
+const jwt_simple = require('jwt-simple');
+const querystring = require('querystring');
+
+const JWT_BEARER_URI = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
+const WELKIN_TOKEN_URL = 'https://api.welkinhealth.com/v1/token';
+const WELKIN_ACCESS_TOKEN = '';
+
+function get_welkin_api_token(client_id, client_secret, scope, endpoint) {
+  const token_data = {
+    'iss': client_id,
+    'aud': endpoint,
+    'exp': Math.round(new Date() / 1000) + 3600, // one hour token request
+    'scope': scope,
+  }
+  const token = jwt_simple.encode(token_data, , 'HS256');
+  try {
+    const res = await axios({
+      method: 'post',
+      url: endpoint,
+      data: querystring.stringify(
+        {'assertion': token.toString(),
+        'grant_type': JWT_BEARER_URI
+        }
+      )
+    });
+    return res.data.access_token;
+  } catch (e) {
+    console.log('error could not get access token');
+    return '';
+  }
+}
+
+WELKIN_ACCESS_TOKEN = get_welkin_api_token(process.env.WELKIN_CLIENT_ID,
+                                           process.env.WELKIN_SECRET,
+                                           'all',
+                                           WELKIN_TOKEN_URL);
+```
+
+```shell
+CURL example not available
 ```
 
-> Example token usage (PYTHON)
+> Example token usage
 
 ```python
+import requests
+
 headers = {"Authorization": "Bearer <token>"}
 
-resp = requests.post('https://api.welkinhealth.com/v1/patients', headers=headers).json()
+resp = requests.get("https://api.welkinhealth.com/v1/patients", headers=headers).json()
+```
+
+```javascript
+const axios = require('axios');
+
+const headers = {'Authorization': 'Bearer <token>'}
+const url = 'https://api.welkinhealth.com/v1/patients';
+
+try {
+  const res = await axios.get(url, {headers: headers});
+  const data = res.data.data;
+} catch (e) {
+  console.log("couldn't get data from Welkin");
+}
+```
+
+```shell
+CURL example not available
```

Comment:
No curl example here?

Reply:
my CURL and bash skills are not good enough to make an JWT in bash :-(